lib/File/Find/Rule.pm

```diff
@@ -485,6 +485,18 @@ sub relative () {
     $self;
 }
 
+=item C<normalize>
+
+Normalize paths found using C<File::Spec->canonpath>
```

Comment:
I agree that behavior is not detailed in the documentation of `canonpath`. I've pushed another commit with some more explanation and an example. Please add to it or change it as you see fit.